src/validators/actix.rs

```diff
@@ -67,9 +67,12 @@ pub async fn clerk_authorize(req: &ServiceRequest, clerk_client: &Clerk) -> Resu
         Err(_) => return Err(HttpResponse::InternalServerError().json("Error: Could not fetch JWKS!"))
     };
     // Parse the request headers
-    let access_token: &str = match req.headers().get("Authorization").unwrap().to_str() {
```

Comment:
this `.unwrap()` would cause service side crash when the user sent the request without the required headers.

Reply:
Great find! Definitely better to throw a more useful error message instead of just crashing on None unwraps. Usually, outputting a 401 is technically more standard since this is authorization middleware. Also, looks like I forgot to run formatting -- I'll make these changes and get this merged!